src/box/lua/config/utils/file.lua

```diff
@@ -55,6 +56,77 @@ local function universal_read(file_name, file_kind)
     return data
 end
 
+
+local function get_file_tags(file_name)
+    local function parse_tags(t)
+        local known_tags = {
+            early_load = 'boolean',
+        }
+
+        local tags = {}
+        for k, v in pairs(t) do
+            if known_tags[k] == type(v) then
+                tags[k] = v
+            end
+        end
+
+        return tags
+    end
+
+    -- Tags are only supported for Lua files.
+    if type(file_name) ~= 'string' or not file_name:endswith('.lua') then
+        error(('Invalid file name %q'):format(file_name), 0)
+    end
+
+    local file, err = io.open(file_name, "r")
+    if file == nil then
+        error(('Unable to open %q: %s'):format(file_name, err), 0)
+    end
+
+    local tags = {}
+    local line = file:read("*line")
```

Comment:
I think we should handle EOF (`nil`) here and below.

Reply:
Thanks for noticing! Fixed!